test/morse_translator_test.rb

```diff
@@ -0,0 +1,16 @@
+require './lib/morse_translator'
+require 'minitest/autorun'
+require 'minitest/pride'
+
+class MorseTransTest < Minitest::Test
+  def test_it_exists
+    morse_translator = MorseTranslator.new
+    assert_instance_of MorseTranslator, morse_translator
+  end
+
+  def test_morse_translate_check
+
+    morse_translator = MorseTranslator.new
+    assert_equal translate, morse_translator.translate("hello world")
```

Comment:
For these tests, generally we want the value that we expect back from the test on the left and the method call on the right. So this test would look something like the following:

```
def test_morse_translator_can_translate
  morese_translator = MorseTranslator.new
  assert_equal "......-...-..--- .-----.-..-..-..", morse_translator.translate("hello world")
end
```

Where did that last line come from? In the spec it says the following in the interaction pattern:

```
pry(main)> morse_translator.translate("hello world")
#=> "......-...-..--- .-----.-..-..-.."
```

So that means to me that when I call the method `translate` with an argument of `"hello world"` that I should get back "......-...-..--- .-----.-..-..-..". In other words, the line on the bottom becomes the thing that I put on the left hand side of the comma, and the line on the top becomes the thing I put on the right hand side of the comma in my `assert_equal` line.